Sofascore/Sofascore/ViewController.swift

```diff
@@ -1,9 +1,60 @@
 import UIKit
+import SofaAcademic
+import SnapKit
 
 class ViewController: UIViewController {
-
+    
+    private let eventRespository = EventRepository(dataSource: MockData())
+    private let leagueView: LeagueView = .init()
+    private let eventStackView: UIStackView = .init()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        addViews()
+        setupConstraints()
+        styleViews()
+        fetchEvents()
+    }
+    
+    func fetchEvents() {
+        eventRespository.fetch { events in
```

Comment:
Mislim da ovdje imas retain cycle, a to si mozes provjeriti u memory graphu (nalazi se pored view hierarchy u xcode-u). Ne znam jeste li spominjali to do sad. Ako niste, komentiraj tu pa ti objasnim.

Reply:
Nismo pa ak mozes pojasnit molim te